Add device specs for GPS data collection (#705)
medic/cht-docs: content/en/core/releases/_index.md

| RAM | 1GB |
| Storage | 8GB |

> If CHWs will be collecting GPS data, the devices should include autonomous GPS sensors in addition to assisted GPS (A-GPS) for areas where GSM network connectivity isn't very good. Autonomous GPS is usually labeled in terms of the supported navigation satellite system:

Reviewer: ```suggestion
If CHWs will be collecting GPS data, autonomous GPS sensors in addition to assisted GPS (A-GPS) for areas with poor GSM network connectivity will enhance the quality of GPS data collected. Autonomous GPS is usually labeled in terms of the supported navigation satellite system:
```
I have rephrased this a bit.
